src/components/CardItem.js

```diff
@@ -1,24 +1,30 @@
 import React, { useState } from "react";
-import styled from "styled-components";
+import styled from "styled-components/macro";
+
+import Heart from "react-animated-heart";
 
 function CardItem(props) {
   const [readMore, setReadMore] = useState(false);
+  const [isClick, setClick] = useState(false);
   return (
     <>
       <Card>
         <CardWrapper>
           <CardImageWrapper data-category={props.label}>
             <CardImage alt="Travel Image" src={props.src} />
+            <CardHeader>{props.header}</CardHeader>
           </CardImageWrapper>
           <CardInfo>
-            <CardHeader className="cards__item__header">
-              {props.header}
-            </CardHeader>
-            <CardText className="cards__item__text">
-              {readMore ? props.text : `${props.text.substring(0, 50)}...`}
+            <CardHeader>{props.header}</CardHeader>
+            <CardText>
+              {readMore ? props.text : `${props.text.substring(0, 25)}...`}
               <ShowText onClick={() => setReadMore(!readMore)}>
                 {readMore ? "show less" : "show more"}
               </ShowText>
+              <Favorite>
+                {props.favorite}
+                <Heart isClick={isClick} onClick={() => setClick(!isClick)} />
```

Comment:
Then here it would be:
```suggestion
                <Heart isClick={isFavorite} onClick={() => setIsFavorite(!isFavorite)} />
```